### PR TITLE
addpkg: cabal-fmt

### PR DESCRIPTION
**Pull request by berberman**
to be reviewed...

### CODE REVIEW

File: archlinuxcn/cabal-fmt/lilac.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+update_on:
+  - source: archpkg
+    archpkg: haskell-optparse-applicative
+  - source: archpkg
+    archpkg: haskell-tasty
+  - source: archpkg
+    archpkg: haskell-tasty-golden
+  - source: hackage
+    hackage: cabal-fmt
```
Comment (lilydjwg) [inline archlinuxcn/cabal-fmt/lilac.yml:12]: Shouldn't this come first (so that `_G.newver` returns the version of it)?

Comment (berberman) [reply archlinuxcn/cabal-fmt/lilac.yml]: Sorry, I thought the order would make no difference...

File: archlinuxcn/cabal-fmt/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# Maintainer: berberman <hatsue@typed.icu>
+
+_hkgname=cabal-fmt
+pkgname=cabal-fmt
+pkgver=0.1.5
+pkgrel=1
+pkgdesc="Format .cabal files"
+url="https://github.com/phadej/cabal-fmt.git"
```
Comment (lilydjwg) [inline archlinuxcn/cabal-fmt/PKGBUILD:8]: This shouldn't be a git URL.

Comment (berberman) [reply archlinuxcn/cabal-fmt/PKGBUILD]: Ah, the author does not provide the field `homepage` as usual, so arch-hs uses https://github.com/phadej/cabal-fmt/blob/dc1e63bb10ddafd6152fae8f7206c3b201b0f7ab/cabal-fmt.cabal#L23 as a fallback.

**Review by lilydjwg — APPROVED** [review]

**Comment by berberman** [comment]
Could you please check this? @felixonmars

**Review by felixonmars — APPROVED** [review]

**Pull request merged**